vendor/modules.txt

```diff
@@ -259,6 +259,7 @@ gopkg.in/tomb.v1
 # gopkg.in/yaml.v2 v2.3.0
 gopkg.in/yaml.v2
 # k8s.io/api v0.19.3 => k8s.io/api v0.19.3
+## explicit
```

Comment:
This might have implications. was this auto-generated or added manually?

Reply:
This was auto generated, I'm not sure what that means actually

Reply:
If I try to manually change it, I get the following error: `k8s.io/api@v0.19.3: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt`

Reply:
Should I leave this in and resolve this?

Reply:
Yes you can resolve